spec/features/admin_series_create_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+describe 'the admin user flow', type: :feature do
+
+  let(:user)  {User.create(email_address: "sam@example.com", admin: true, password: "password")}
+  let(:subscription)  {Subscription.create(email_address: "coll@example.com", name: "Colleen")}
+
+  before(:each) do
+    series = Series.create(name: "cheers", seasons: 1)
+    episode = Episode.create(summary: "lorem ipsum", season: 1)
+    series.episodes << episode
+    subscription.series << series
+    user.save
+    subscription.save
+    visit login_path
+    page.fill_in('email_address', with: 'sam@example.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.